tjpcov/covariance_fourier_cNG.py

```diff
@@ -0,0 +1,238 @@
+import os
+
+# import healpy as hp
+import numpy as np
+import pyccl as ccl
+
+from .covariance_builder import CovarianceFourier
+
+
+class FouriercNGHaloModel(CovarianceFourier):
+    """Class to compute the CellxCell Halo Model cNG Covariance."""
+
+    cov_type = "cNG"
+
+    def __init__(self, config):
+        """Initialize the class with a config file or dictionary.
+
+        Args:
+            config (dict or str): If dict, it returns the configuration
+                dictionary directly. If string, it asumes a YAML file and
+                parses it.
+        """
+        super().__init__(config)
+
+        self.cNG_conf = self.config.get("cNG", {})
+
+        self.HOD_dict = {
```

Comment:
We need to homogenize this with all the other halo model things in the SSC. We can do it in another pull request